reference/customers.v2.yml

```diff
@@ -832,26 +840,26 @@ paths:
       tags:
         - Customer Groups
       summary: Get All Customer Groups
-      description: 'Returns a list of *Customer Groups*. Default sorting is by customer-group id, from lowest to highest.'
+      description: Returns a list of *Customer Groups*. Default sorting is by customer-group ID, from lowest to highest.
       operationId: getAllCustomerGroups
       parameters:
         - name: page
           in: query
-          description: Number of pages
+          description: Number of pages.
           schema:
             exclusiveMaximum: false
             exclusiveMinimum: false
             type: number
         - name: limit
           in: query
-          description: Count per page
+          description: Count per page.
           schema:
             exclusiveMaximum: false
             exclusiveMinimum: false
             type: number
         - name: name
           in: query
-          description: 'Filter customer groups by exact name match. Can use `name:like` to filter using a fuzzy matching method. This is good for implementing search.'
+          description: Filter customer groups by exact name match. Can use `name:like` to filter using a fuzzy matching method. This is good for implementing search.
```

Comment:
add `name:like`. I don't see it here.

Reply:
After testing the description could be clearer. Like isn't a param but can be added to the `name:` param to act like a search filter.